docs/tutorials/connect-integration.rst

```diff
@@ -0,0 +1,112 @@
+        INSERT 0 1
+        postgres=# INSERT INTO users (username, popularity) VALUES ('user2', 5);
+        INSERT 0 1
+        postgres=# INSERT INTO users (username, popularity) VALUES ('user3', 75);
+        INSERT 0 1
+
+When you are done, you can remove this information by ``docker kill some-postgres && docker rm some-postgres``.
+
+Create a JDBC Source Connector
+------------------------------
+
+Now that Postgres is up and running with a database for your user, you can connect to it via KSQL.
+If you are using the default configurations, it will connect automatically to your connect cluster.
+Otherwise, you will need to change the ``ksql.connect.url`` to point to your connect deployment.
+
+.. code::
+
+    ksql> CREATE SOURCE CONNECTOR `jdbc-connector` WITH(\
+      "connector.class"='io.confluent.connect.jdbc.JdbcSourceConnector',\
+      "connection.url"='jdbc:postgresql://localhost:5432/YOUR_USERNAME',\
+      "mode"='bulk',\
+      "topic.prefix"='jdbc-',\
+      "key"='username');
+
+Profit
+------
+
+At this point, data should automatically start flowing in from Postgres to KSQL! You can confirm this
+by running ``DESCRIBE CONNECTOR "jdbc-connector";``. Your output should resemble:
+
+.. code::
+
+    Name                 : jdbc-connector
+    Class                : io.confluent.connect.jdbc.JdbcSourceConnector
+    Type                 : source
+    State                : RUNNING
+    WorkerId             : 10.200.7.69:8083
+
+     Task ID | State   | Error Trace
+    ---------------------------------
+     0       | RUNNING |
+    ---------------------------------
+
+     KSQL Source Name     | Kafka Topic | Type
+    ---------------------------------------------
+     JDBC_CONNECTOR_USERS | jdbc-users  | TABLE
+    ---------------------------------------------
+
+Now select everything from the topic to see what is going on and how it gets auto populated:
+
+.. code::
+
+    ksql> SELECT * FROM JDBC_CONNECTOR2_USERS;
+
+    +------------------+------------------+------------------+------------------+
+    |ROWTIME           |ROWKEY            |USERNAME          |POPULARITY        |
+    +------------------+------------------+------------------+------------------+
+    |1566336783102     |user1             |user1             |100               |
+    |1566336783102     |user2             |user2             |5                 |
+    |1566336783102     |user3             |user3             |75                |
+    |1566336788106     |user1             |user1             |100               |
+    |1566336788106     |user2             |user2             |5                 |
+    |1566336788106     |user3             |user3             |75                |
+
+You'll notice that the users are repeated multiple times. This is because we specified ``bulk``
+as the mode, which re-imports the entire database every time. This is obviously not fit for
+production - see `Incremental Query Modes
```

Comment:
```suggestion
production. For more information on changelog capture, see :ref:`jdbc-source-connector-incremental-query-modes`.
```

Reply:
The `jdbc-source-connector-incremental-query-modes` anchor doesn't exist yet and must be added to the kafka-connect-jdbc source index. Ping me when you're ready, and I can add it for you.